File: Topic Based Problems/01 Fundamentals/1 Beginner/07 Circle Properties/C++/CircleProperties.cpp
#include <iostream>
#include <cmath>

#define M_PI 3.14159265358979323846

using namespace std;

int main() {
    double radius;

    cout << "Enter radius: ";
    cin >> radius;

    cout << "Diameter of circle is: " << 2 * radius << endl;
    cout << "Circumference of circle is: " << 2 * M_PI * radius << endl;
    cout << "Area of circle is: " << M_PI * pow(radius, 2) << endl;

    return 0;
}

File: Topic Based Problems/01 Fundamentals/1 Beginner/07 Circle Properties/Java/CircleProperties.java
import java.util.Scanner;

public class CircleProperties {
    public static void main(String[] args) {
        double radius;

        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter radius: ");
        radius = scanner.nextDouble();

        System.out.println("Diameter of circle is: " + (2 * radius));
        System.out.println("Circumference of circle is: " + (2 * Math.PI * radius));
        System.out.println("Area of circle is: " + (Math.PI * Math.pow(radius, 2)));

        scanner.close();
    }
}
